README.md

```diff
@@ -18,29 +48,16 @@ xrootdredirector = cms-xrd-global.cern.ch
 
 ```
 
-In case you update module, you should reinstall the packages dependencies using:
-
-```sh
-pip install --upgrade .
+## Running the framework
```

Comment:
add info about running source setup.sh in order to start using the framework?

Reply:
Absolutely; this is a crucial step that I forgot to mention.